app/app.ts

```diff
@@ -129,6 +129,7 @@ class USMApp {
                 $logProvider.debugEnabled(true);
             }])
             .config(['RestangularProvider', function(RestangularProvider) {
+                RestangularProvider.setBaseUrl('http://api.tendrl.org/1.0/');
```

Comment:
1. It will be config param in next up patches, as mentioned in #27 (and in description) for the moment, it's closest to the same code it used to be (so that it doesn't break anything)
2. It can't be relative path (in it's true form) unless the API and frontend are hosted in the same server & port, which is not the case.